test/formats/common.py

```diff
@@ -52,7 +55,17 @@ def save_and_load_metadata(filename, metadata):
 
 
 def load_raw(filename):
-    return mutagen.File(filename)
+    file = mutagen.File(filename)
```

Comment:
Can we rename this? File is a keyword.

Reply:
Good point. I addressed this for `load_raw` / `save_raw` in this file. But this is actually something we should look into over the entire source. A few places use `file_` or just `f`, but usage of `file = ...` is quite common in the codebase.